### ISSUE TITLE
Identity: name, graphic choices

### ISSUE THREAD

**Issue by raphaelbastide** [bpierre/gtranslate#27]
## Name
The current name “gTranslate” refers to Google translation service. As the translation API [may change](https://github.com/bpierre/gtranslate/issues/23#issuecomment-91146508) in a few versions, it may be time to call this dependency into question, and consequently to change the name. I believe that the name can be more precise and related to the extension function: easily accessible, right next to the mouse, contextual translation. I don’t have any ideas but I think @bpierre does. I can think about it as well.
By the way, Gtranslate is also a [Joomla extension](http://extensions.joomla.org/extension/languages/automatic-translations/gtranslate).

## Icon
Currently, the gtranslate icon is a white “a” on a purple background. I think it is too far from the idea of translation and too far from the name too. If the name comes to change, This icon have to change too. We could imagine a simple symbol, related to the name (and chose a name that can be easily symbolised as well!).

## Consequences
I wonder what are the consequence for a name refresh, for the [extension page](https://addons.mozilla.org/en-US/firefox/addon/gtranslate/) and for the users. Is it a big deal, should we make a public announcement or communicate this change? If yes, where?

Let me know what you think about those possible changes and I will make a proposal if I have a “go”.

**Comment by bpierre** [comment]
> The current name “gTranslate” refers to Google translation service.

I agree that the name relates to Google, and maybe a change is required, especially if we introduce other translation services. But maybe it’s not necessary, the “g” in gtranslate could also mean “good”, “great” or anything else.

Suggestions, right off the top of my head (I don’t really like any of them):

- Right Translate (right as in right click and as in “correct”)
- Menu Translate
- xtranslate or xTranslate (x as in “any provider”)

>  By the way, Gtranslate is also a Joomla extension.

I don’t think this is an issue, gtranslate is still easily findable, either with “gtranslate”, or “gtranslate firefox” and “gtranslate addon”.

> Currently, the gtranslate icon is a white “a” on a purple background. I think it is too far from the idea of translation and too far from the name too. If the name comes to change, This icon have to change too.

I agree about the need to change the icon, even if the name does not change.

> I wonder what are the consequence for a name refresh, for the extension page and for the users. Is it a big deal, should we make a public announcement or communicate this change? If yes, where?

Technically it looks feasible on addons.mozilla.org, without having to create a new addon. But as you said, people know and rely on the name to find gtranslate. This could be mitigated though, for example by adding “Formerly known as gtranslate” in the description and the tags.

**Comment by mote0230** [comment]
I'm not a big proponent of a name change when not absolutely necessary, the current name is, by its long lifetime and large userbase already all over the place, in a lot of forums where people recommend it and in current users their memory. In my experience a lot of non-IT savvy users are easily confused, and I imagine a name change would do more bad than good. 

As long as google has the best translations to offer, people will want to use google, and if we stop offering it some users might change to a different addon. On the positive side, as long as the google translate website works, there will be some way to make similar requests.

**Comment by raphaelbastide** [comment]
Some tests for an icon. Ignore the color for now. Those are just concept drafts to open a discussion about sign and typography. Let me know what you think!

![test](https://cloud.githubusercontent.com/assets/193122/7784286/aead1c12-0160-11e5-9a84-8e1a30ef8379.png)

**Comment by bpierre** [comment]
Quick feedback, sorry I am quite busy these days:

- I like them all, the G and the arrow are both identifiable and precise
- Maybe the first one is too similar to the Google favicon?
- I think the third one is interesting especially as a list bullet icon, I wonder how well it could work as a context menu icon
- I think the third one is interesting especially as a list bullet icon, I wonder how well it could work as a context menu icon

**Comment by bpierre** [comment]
I forgot: :+1: :cool: :tada:

**Comment by raphaelbastide** [comment]
Here is a new test, with lower-case “g” (because the exact name is “gtranslate”). Additionally, I tried to refer to the top of a “g” with a loop-like shape.

![export2](https://cloud.githubusercontent.com/assets/193122/7840442/a26098c8-0497-11e5-8945-77959ab4220f.png)
